src/viewer.js

```diff
@@ -2042,10 +2042,6 @@ class VolumeImageViewer {
 
     const dragPanOptions = {
       features: this[_features],
-      condition: event => {
```

Comment:
Not clear why the condition existed at all - seems to just need to be deleted.
@igoroctaviano  - can you comment on this?